tests/playbooks/kubevirt_pvc_img_upload.yml

```diff
@@ -37,11 +37,13 @@
       delay: 10
       retries: 30
 
+    # You need to be able to connect to the cdi upload proxy specified in `upload_host` for this to work.
+    # There's info in README.md on how to do that.
```

Comment:
I would replace this with: `More information can be found in README.md`

Reply:
done